src/app/zones/views/ZoneCreateView.vue

```diff
@@ -277,7 +283,7 @@ async function createZone() {
   try {
     zone.value = await kumaApi.createZone({ name: name.value })
   } catch (err) {
-    if (err instanceof ApiError && HANDLED_STATUS_CODES.includes(err.status)) {
+    if (err instanceof ApiError && [409, 500].includes(err.status)) {
```

Comment:
nitpick: why did you inline it? could've just been changed at the 241 line.

Reply:
I figured it’s better to have this inline as the constant wasn’t re-used and it doesn’t really aid in the readability here. It’s only two values now.